index.html

```diff
@@ -29,5 +29,111 @@ <h4>let's connect</h4>
             <li><a href="#"><img src="./images/Vector3.svg" alt=""></a></li>
         </ul>
     </main>
+
+    <section class="grid-container">
+        <section class="skills">
```

Comment:
[OPTIONAL] Kindly replace the `<section>` tag with `<div>` since these are Projects and not sections of the page.

- _A section represents a generic standalone section of the document, so use it. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section) for more info._

Reply:
@awais-amjed I used `<div>` previously. I was asked to change it to `<section>`. Which do I follow now. This is my third and finally review.

Reply:
@Allenkeys `section` is for the whole block as @levy002 mentioned before. But the individual projects should be `div`. I hope you understand 😃

Reply:
Implemented 👍